tests/sles4sap/robot_fw.pm

```diff
@@ -17,6 +17,7 @@ use testapi;
 use strict;
 use warnings;
 use version_utils 'is_sle';
+use utils "zypper_call";
```

Comment:
As far as I remember @alvarocarvajald said that we should now use `qw( ... )` in all cases, no?

Reply:
I don't remember, I can quickly change it anyway.

Reply:
Yeah, more info in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9931

Reply:
Done